services/table-foreign-keys-analyzer.js

```diff
@@ -8,19 +8,41 @@ function TableForeignKeysAnalyzer(databaseConnection, schema) {
     switch (queryInterface.sequelize.options.dialect) {
       case 'postgres':
```

Comment:
It is done only for postgres 🤔  ?

Reply:
I will push the other scripts one by one